guides/common/modules/con_prerequisites-for-configuring-smart-proxy-server-for-load-balancing.adoc

```diff
@@ -1,16 +1,4 @@
 [id="Prerequisites_for_Configuring_{smart-proxy-context}_Servers_for_Load_Balancing_{context}"]
 = Prerequisites for configuring {SmartProxyServers} for load balancing
 
-ifdef::orcharhino[]
 You can find a list of requirements for {SmartProxyServer} in xref:sources/installation_and_maintenance/installing_orcharhino_proxy.adoc[_{InstallingSmartProxyDocTitle}_].
```

Comment:
@maximiliankolb I noticed you're doing things differently for this part of the docs for orcharhino so I left this file in just for orcharhino. (I moved the conditionals from here to LB/master.adoc).